.travis.yml

```diff
@@ -4,3 +4,11 @@ dart:
   - dev
   - stable
 script: ./tool/travis.sh
+
+# Only building master means that we don't run two builds for each pull request.
+branches:
+  only: [master]
```

Comment:
We can always add it back – although it just doubles the runtime if it's going to end up being a PR, too.